internals/models.py

```diff
@@ -825,6 +828,32 @@ def get_feature(self, feature_id, update_cache=False):
 
     return feature
 
+  @classmethod
+  def get_by_ids(self, feature_ids, update_cache=False):
+    result_dict = {}
+    futures = []
+
+    for feature_id in feature_ids:
+      KEY = '%s|%s' % (Feature.DEFAULT_CACHE_KEY, feature_id)
+      feature = ramcache.get(KEY)
+
+      if feature is None or update_cache:
+        futures.append(Feature.get_by_id_async(feature_id))
```

Comment:
Does `Feature.get_by_id_async` update `ramcache` automatically? (nm, it is cached below)

Reply:
No action if `feature is not None`?

Reply:
Right.  If it is not none, then that means that we got a ramcache hit on line 838.